casepropods/family_connect_registration/plugin.py

```diff
@@ -1,13 +1,51 @@
+from confmodel import fields
+from casepro.cases.models import Case
 from casepro.pods import Pod, PodConfig, PodPlugin
+import requests
 
 
 class RegistrationPodConfig(PodConfig):
-    pass
+    url = fields.ConfigText("URL to query for the registration data",
+                            required=True)
+    token = fields.ConfigText("Authentication token for registration endpoint",
+                              required=True)
+    field_mapping = fields.ConfigList(
+        "Mapping of field names to what should be displayed for them",
```

Comment:
I think just adding the example to the doc string of the field should be enough. You can do multiline doc strings like this: https://github.com/praekelt/vumi/blob/develop/vumi/transports/httprpc/httprpc.py#L48-L54